src/core/Utils.js

```diff
@@ -367,7 +367,7 @@ function copyToClipboard (textToCopy, fallbackString) {
     const magicCopyFailed = (err) => {
       document.body.removeChild(textArea)
       window.prompt(fallbackString, textToCopy)
-      return reject('Oops, unable to copy displayed fallback prompt: ' + err)
+      return reject(new Error('Oops, unable to copy displayed fallback prompt: ' + err))
     }
```

Comment:
> unable to copy displayed fallback prompt

does this mean “unable to copy, so displayed fallback prompt instead”? :) should it reject if the prompt is successful shown?

Reply:
yeah, figure this should never have to reject. thanks!